bee-message/src/output/cpt2/signature_locked_dust_allowance.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    address::Address,
+    constant::{DUST_DEPOSIT_MIN, IOTA_SUPPLY},
+    Error,
+};
+use packable::{bounded::BoundedU64, Packable};
+use std::ops::RangeInclusive;
```

Comment:
```suggestion

use std::ops::RangeInclusive;
```

FYI we put a new line between groups of imports
- self crate
- bee crates
- 3rd parties
- std/core

Reply:
Your ways are strange to me, magic man.